src/api/spec/jobs/report_to_scm_job_spec.rb

```diff
@@ -0,0 +1,102 @@
+require 'rails_helper'
+
+RSpec.describe ReportToScmJob, vcr: false do
+  let(:user) { create(:confirmed_user, login: 'foolano') }
+  let(:token) { Token::Workflow.create(user: user, scm_token: 'fake_token') }
+  let(:project) { create(:project, name: 'project_1', maintainer: user) }
+  let(:package) { create(:package, name: 'package_1', project: project) }
+  let(:repository) { create(:repository, name: 'repository_1', project: project) }
+  let(:event) { Event::BuildSuccess.create({ project: project.name, package: package.name, repository: repository.name, reason: 'foo' }) }
```

Comment:
This event creation triggers a `ReportToScmJob` run because [this after_create](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/event/base.rb#L166) and [our current](https://github.com/openSUSE/open-build-service/blob/master/src/api/config/environments/test.rb#L63) active job configuration.